code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -30,6 +30,7 @@
 	var/list/hands_nodrop = list()
 	var/obj/item/clothing/head/helmet/space/chronos/helmet = null
 	var/obj/effect/chronos_cam/camera = null
+	var/mutable_appearance/phase_underlay = null
```

Comment:
```suggestion
	var/mutable_appearance/phase_underlay
```

Reply:
phase_underlay is now so null it doesn't even exist